package.json

```diff
@@ -8,7 +8,7 @@
   },
   "main": "index.js",
   "scripts": {
-    "deploy": "gcloud app deploy",
+    "deploy": "node create_secrets.js && gcloud app deploy",
```

Comment:
Ah, so if there are dependencies needed for deploy, try using gcp-build script:
https://cloud.google.com/appengine/docs/standard/nodejs/running-custom-build-step

Real example in https://github.com/foolip/mdn-bcd-collector/blob/master/package.json.

Reply:
I think I did it right...